omero/sysadmins/omeroweb-upgrade.rst

```diff
@@ -29,19 +29,19 @@ satisfied all the :doc:`system requirements <system-requirements>` with
 Configuration
 ^^^^^^^^^^^^^
 
-This documentation assumes that OMERO.web is deployed **separately** from OMERO.server.
-This is the recommended installation set-up as they
-perform best under different circumstances and require a different set of
-dependencies. Please check :ref:`omero_web_deployment` for the latest advice
```

Comment:
Note sure that we should remove this link, but leaving it to others to ponder since there's enough here that needs merging.